### PR TITLE
Gemfile and .lock with default plugins

### LINKED ISSUE #2733: Gemfile & Gemfile.jruby-1.9.lock for 1.5-rc1 should contain default plugins
make sure 1.5-rc1 branch/tag contains Gemfile & Gemfile.jruby-1.9.lock with all packaged plugins. This is really important to be able to recreate the package with the correct plugins versions originally included in the released package.

### PR DESCRIPTION
**Pull request by colinsurprenant**
`Gemfile` and `Gemfile.jruby-1.9.lock` with:
```sh
rake bootstrap
rake plugin:install-defaults
```

fixes #2733

### CODE REVIEW

File: Gemfile.jruby-1.9.lock [hunk 1]
```diff
@@ -126,12 +538,260 @@ GEM
...
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot
+      polyglot (>= 0.3.1)
```
Comment (ph) [inline Gemfile.jruby-1.9.lock]: too much polyglot here?

Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock]: we don't control that. bug in Bundler.

Comment (ph) [inline Gemfile.jruby-1.9.lock]: Which version of bundler do you have? I though a recent version fixed this issue?

Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock]: ah good point, bundler (1.7.4), upgrading to 1.8.3 and rebuilding.

Comment (purbon) [inline Gemfile.jruby-1.9.lock]: as @ph said recent version fix that ... at least for me.

Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock]: but wait, it should use the bundler installed in vendor/bundler which is the last version... !?

Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock]: yup. did not change, it's using Bundler 1.8.3 that we are installing.

Comment (ph) [inline Gemfile.jruby-1.9.lock]: Weird, https://github.com/bundler/bundler/issues/3216 seems to be fixed in 1.7.9?

Comment (colinsurprenant) [inline Gemfile.jruby-1.9.lock]: Don't know... and I don't have time to check this now, this is harmless so I will not try to fix it now but please go ahead if you want to try to fix it!

**Comment by ph** [comment]
LGTM, the polyglot is harmlesss.
I've run the task I have the polyglot issue could be a regression on their side.

**Comment by suyograo** [comment]
Merged to 1.5 https://github.com/elasticsearch/logstash/commit/909778a75fac92248466347e8d0269daa7c7c1ec

**Pull request closed without merging**